webrender/src/resource_cache.rs

```diff
@@ -709,7 +709,7 @@ impl ResourceCache {
                         is_opaque: image_descriptor.is_opaque,
                     }
                 } else {
-                    image_template.descriptor.clone()
+                    image_template.descriptor
```

Comment:
This change is not related to what the PR describes, please remove of put in its own commit.